docs/source/gdpr/i18n/data_user_agreement.es.rst

```diff
@@ -2,20 +2,20 @@
 
 Spanish
 -------
-(translation courtesy of Dr Maria de la Iglesia)
+(translation courtesy of Dr Maria de la Iglesia and Guiomar Niso)
 
-Acuerdo de usuario de datos para acceder a datos humanos identificables
+Acuerdo de Uso de Datos para el acceso a datos humanos identificables
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**Version:** OBC-GDPR-DUA.es 1.0.0
+**Version:** OBC-GDPR-DUA.es 1.0.1
```

Comment:
@guiomar does this reflect change or overall corresponds to the change in 
https://github.com/con/open-brain-consent/pull/116/files

?

if yes -- we could keep this version, but if somehow it reflects prior broken version more (very unlikely, dunno how, but want to make sure), we might want to do smth like

```suggestion
**Version:** OBC-GDPR-DUA.es 1.0.0+es1
```

or alike to signify that.